lib/lita-reviewme/github.rb

```diff
@@ -0,0 +1,66 @@
+require 'delegate'
+
+module Lita
+  module Reviewme
+    class Github
+      extend Forwardable
+
+      attr_reader :config, :repo, :pr_id
+
+      delegate [
+        :github_access_token,
+        :github_comment_template,
+        :github_comment,
+        :github_request_review
+      ] => :config
+
+      def initialize(config, repo, pr_id)
+        @config = config
+        @repo = repo
+        @pr_id = pr_id
+      end
+
+      def client
+        @client ||= Octokit::Client.new(access_token: github_access_token)
+      end
+
+      def pull_request
+        @pull_request ||= client.pull_request(repo, pr_id)
+      end
+
+      def owner
+        pull_request.user.login
+      rescue Octokit::Error
+        raise UnknownOwner
+      end
+
+      def assign_and_or_comment(reviewer)
```

Comment:
Maybe this is a leaky abstraction. I'd probably just name the method `assign` by the notion that the library may be configured to "assign" by "adding a comment" or "assigning gh reviewer" or "both" or "neither".

Speaking of neither, should be emit a warning if they decide to set comment and gh-review to false?